src/replaceCustomValues.ts

```diff
@@ -440,6 +440,14 @@ function formatValue({
       }
       return undefined
     }
+    case 'number': {
```

Comment:
Can this be the same case as `calculation` now?

Reply:
nyers